pkg/crc/preflight/preflight_tray_powershell_windows.go

```diff
@@ -0,0 +1,170 @@
+package preflight
+
+import (
+	"fmt"
+	"strings"
+)
+
+var (
+	trayInstallationScript = []string{
```

Comment:
@anjannath why don't you use something like so you don't need to use a different function just to join them with a new line?
```golang
taryInstallationScriptTemplate = `$ErrorActionPreference = "Stop"
$password = "%s"
$tempDir = "%s"
...
`
```